src/app/misc/Feedback.tsx

```diff
@@ -0,0 +1,88 @@
+import { IconButton } from '@chakra-ui/button';
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverHeader,
+  PopoverBody,
+  Portal,
+  Center,
+  Text,
+  Box,
+  Link,
+  VStack,
+  Image,
+} from '@chakra-ui/react';
+import { PropsWithChildren, useState } from 'react';
+import { VscFeedback, VscChevronDown } from 'react-icons/vsc';
+
+type Props = {
+  src: string;
+  href: string;
+};
+
+function FeedbackElement({ href, src, children }: PropsWithChildren<Props>): JSX.Element | null {
+  return (
+    <Link href={href} target="_empty" _hover={{ textDecoration: 'none' }}>
+      <VStack borderRadius="xl" bg="white" mt="10px" shadow="md">
+        <Center bg="blue.300" borderTopRadius="xl" maxH="60px" objectFit="cover" overflow="hidden">
+          <Image src={src} h="100%" objectFit="cover" overflow="hidden" />
+        </Center>
+        <Box px="20px" pb="20px">
+          <Text>{children}</Text>
+        </Box>
+      </VStack>
+    </Link>
+  );
+}
+
+export function Feedback(): JSX.Element | null {
+  const [isOpen, setIsOpen] = useState(false);
+  const open = () => setIsOpen(!isOpen);
+
+  return (
+    <Popover closeOnEsc={false} isOpen={isOpen} autoFocus={false}>
```

Comment:
why not closeOnEsc? I feel like the popover should close if someone clicks outside of it also, maybe...?

Reply:
Copied this website pretty closely https://www.intercom.com/
They didn't have closeOnEsc but i'm indifferent